packages/@okta/vuepress-site/guides/sign-into-spa/use-the-access-token/vue/getaccesstoken.md

```diff
@@ -0,0 +1,11 @@
+Get the access token using the [getAccessToken()](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-vue#authgetaccesstoken) method on the [auth](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-vue#auth) object object. Then, use this value to add an `Authorization` header to outgoing requests:
```

Comment:
Get the access token using the [getAccessToken()](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-vue#authgetaccesstoken) method on the [auth](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-vue#auth) object. Then, use this value to add an `Authorization` header to outgoing requests: